### ISSUE TITLE
Trouble downloading mods using nxm links

### ISSUE THREAD

**Issue by WGuptill** [ModOrganizer2/modorganizer#877]
## The problem:
Whenever I try to download a mod (from any browser) while MO2 is running, it crashes with the error message "failed to recieve data from secondary instance: unknown error" (this message pops up maybe 5 times or so before MO2 gives up and closes itself down)

## To Reproduce:
1. Start MO2
2. Go to a mod page
3. Click on nxm download link
4. Profit?? (This is where it crashes)

## Environment:
* Mod Organizer v2.2.1
* Windows 10 pro OS
* Any Browser

## Details:
![image](https://user-images.githubusercontent.com/51980735/67128270-cd2ffd80-f1c0-11e9-9d95-3952cb34852d.png)
As shown above, when trying to download a mod with MO2 already open, I get the error. However, if I start a nxm link download while MO2 is closed, it goes through just fine. This error started on Mozilla Firefox, so I switched over to Chrome and it worked for a while before starting to have the error again. I tried both edge and (I'm sorry) Explorer, and neither of them were fruitful. I have tried reinstalling MO2 to no avail.

## Link to Mod Organizer logs:
[Logs.zip](https://github.com/ModOrganizer2/modorganizer/files/3745662/Logs.zip)

### USVFS:
https://gist.github.com/WGuptill/3a1cde5baf71bf8dbe9179d9eb47fd9d

### MO Interface:
https://gist.github.com/WGuptill/ba36f16be914fd5b890484acf2d5d275

**Comment by LostDragonist** [comment]
This is a bit of a known issue with no known cause or solution.  The error is truly unknown and the documentation for the functions being used say they can more or less fail randomly.  Fixing this issue for good would require some serious architecture changes in the way downloads are handled, which probably isn't happening any time soon.

The only advice I can really give is to slow down when downloading multiple files in a row.

**Comment by LostDragonist** [comment]
For what it's worth, you do have a couple programs going on that could possibly make the issue worse, maybe...?  Kapersky and McAfee are always somewhat questionable to see.  ASUS Sonic Radar is known to interfere with some MO GUI stuff.

**Comment by WGuptill** [comment]
I added exceptions for MO2 in Kapersky and McAfee and it seems to be working for the time being. Thanks for letting me know about that.

**Comment by givemeyoureyes** [comment]
I"m having the exact same problem. Just downloaded MO2 for the first time - version 2.2.1. Was going through the Tutorial, went to nexus to download my first mod per the tutorial's instructions. I'm on Chrome FYI. Click to download, accept the 'Open NXM Link Proxy' pop-up and then get the "failed to receive data from secondary instance:unknown error" message.

Tried it with MO2 closed as WGuptill suggested and it goes through just fine.

A fix would be great, but I'll get by for now...

**Comment by WGuptill** [comment]
givemeyoureyes, I found a permanent fix. Switch to Vortex. No problems.

**Comment by Al12rs** [comment]
Duplicate of #884
